src/store/gcodePreview/getters.ts

```diff
@@ -48,21 +48,31 @@ export const getters: GetterTree<GcodePreviewState, RootState> = {
       ? (a: number, b: number) => Number.isNaN(a) || a < b
       : (a: number, b: number) => a !== b
 
-    for (let index = 0; index < moves.length; index++) {
-      if (moves[index].z !== undefined && z !== moves[index].z) {
-        z = moves[index].z
+    moves.forEach((move, index) => {
+      if (move.z !== undefined && z !== move.z) {
+        z = move.z
         zStart = index
       }
 
-      if (moves[index].e > 0 && zCmp(zLast, z)) {
+      if (move.e > 0 && zCmp(zLast, z)) {
         zLast = z
 
         output.push({
           z,
           move: zStart,
-          filePosition: moves[index].filePosition
+          filePosition: move.filePosition
         })
       }
+    })
```

Comment:
This is largely just noise. I refactored the `for` loop to a `forEach` but functionality remains unchanged

Reply:
Can't remember what it was specifically, but I think I've had issues doing that in the past... 🤔